test/src/windows/portableTest.ts

```diff
@@ -54,4 +54,16 @@ test.ifNotCiMac("portable - artifactName and request execution level", app({
   projectDirCreated: projectDir => {
     return copyTestAsset("headerIcon.ico", path.join(projectDir, "build", "foo test space.ico"))
   },
-}))
\ No newline at end of file
+}))
+
+test.ifNotCiMac("portable - splashImage", app({
+  targets: Platform.WINDOWS.createTarget(["portable"]),
+  config: {
+    publish: null,
+    portable: {
+      //tslint:disable-next-line:no-invalid-template-strings
+      artifactName: "${productName}Portable.${version}.${ext}",
+      splashImage: path.resolve(getFixtureDir(), "installerHeader.bmp"),
```

Comment:
I am not a fan of this one. However, I tried tracing through the code quickly and did not find an appropriate location to translate relative input (`some-dir/my-image.bmp`) to the full path (`C:\\Users\\Me\\my-project\\some-dir\\my-image.bmp`). I am not sure if NSIS will use a relative path (most likely yes), but it didn't work when I just followed the examples of existing tests.

I'd appreciate any help here. Where would be the best place in the code to resolve the path for this image?